src/components/organims/PrivacyBox.js

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import Auth0Lock from "auth0-lock";
+import { ApolloClient } from 'apollo-client';
+import { ApolloProvider } from 'react-apollo';
+import { InMemoryCache } from 'apollo-cache-inmemory';
+
+import PrivacyLogin from './privacy/PrivacyLogin';
+import Offboarding from './privacy/Offboarding';
+import OffboardingCancel from './privacy/OffboardingCancel';
+import OffboardingComplete from './privacy/OffboardingComplete';
+import OffboardingFail from './privacy/OffboardingFail';
+
+import { httpLink, authLink } from '../../utils/apollo';
+
+class PrivacyBox extends React.Component {
+  constructor(props) {
+    super(props);
+
+    // Will hold Apollo Client
+    this.client = null;
+    // Init Auth0 Lock
+    this.lock = this.initLock();
+
+    // Bind necessary functions
+    this.onLogin = this.onLogin.bind(this);
+    this.onLogout = this.onLogout.bind(this);
+    this.navigate = this.navigate.bind(this);
+    this.handleAuthentication = this.handleAuthentication.bind(this);
+
+    // Init internal State
+    this.state = {
+      isAuthenticated: false,
+      profile: null,
+      route: null,
+    };
+  }
+  
+  initLock() {
```

Comment:
Would have kept the initialilization of auth0Lock closer to `LoginBox` (ie in `LoginBox`)

Reply:
I agree it would have been better if isolated into its own component (`PrivacyLogin` I think, as `LoginBox` is just a stylized div to insert the lock DOM).